src/main/ts/Utils.ts

```diff
@@ -17,33 +16,35 @@ const isEventProp = (name: string): name is keyof IEventPropTypes => {
   return name in eventPropTypes;
 };
 
-interface EventHandlerSet {
-  handler: EventHandler<unknown>;
-  eventName: string;
+const eventAttrToEventName = <T extends string>(attrName: `on${string & T}`): T => {
```

Comment:
Do you need `string & T` here? Would `T` suffice?

Reply:
It's what they do in the examples but I didn't try it.

Reply:
I tried it without the `string &` and it seems to still work so I removed it.